x-pack/plugins/cases/common/schema/index.test.ts

```diff
@@ -281,4 +287,26 @@ describe('schema', () => {
           `);
     });
   });
+
+  describe('limitedNumberSchema', () => {
```

Comment:
Can you add a test where tests that the validation succeeds if it in between min and max?